.github/workflows/regression-tests.yml

```diff
@@ -165,7 +165,7 @@ jobs:
       run: conda install --yes -c msys2 m2-base m2-make m2w64-toolchain libpython
     - name: Install cocotb runtime dependencies (Windows, msvc)
       if: startsWith(matrix.os, 'windows') && matrix.toolchain == 'msvc'
-      run: conda install --yes -c msys2 m2-base m2-make
+      run: conda install --yes -c msys2 m2-base m2-make sqlite
```

Comment:
Maybe this?

```suggestion
      run: conda install --yes -c msys2 m2-base m2-make sqlite=3.39
```

Reply:
Appreciate the suggestion, it's still broken